eng/common/pipelines/templates/steps/detect-api-changes.yml

```diff
@@ -0,0 +1,31 @@
+parameters:
+  ArtifactPath: $(Build.ArtifactStagingDirectory)
+  Artifacts: []
+
+steps:
+  - pwsh: |
+      $artifacts = '${{ convertToJson(parameters.Artifacts) }}' | ConvertFrom-Json
```

Comment:
Why create the list here if you are just going to parse it again below I gave you as suggestion below that might work.

Reply:
Done